packages/crud/src/vaadin-crud-dialog.js

```diff
@@ -33,6 +33,24 @@ registerStyles(
       flex: none;
       flex-direction: row-reverse;
     }
+
+    :host([fullscreen]) {
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      padding: 0;
+    }
+
+    :host([fullscreen]) [part='overlay'] {
+      height: 100vh;
+      width: 100vw;
```

Comment:
I wonder if these will cause unwanted scrollbars in some situations? Any reason why we don't use `100%`?

Reply:
Well, I see they have been like this before, so I suppose it’s fine as it is.

Reply:
I was also suspicious about this, but probably we can change this separately.